src/unix/linux-core.c

```diff
@@ -193,9 +193,11 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
   int count;
   int nfds;
   int fd;
-  int op;
+  int op = 0;
   int i;
 
+  memset(&e, 0, sizeof(struct uv__epoll_event));
```

Comment:
As i said before, not initializing those structs gives me `unitialized values` warnings in valgrind.